pkg/cmd/opts/gocloud_integration_test.go

```diff
@@ -0,0 +1,36 @@
+package opts_test
```

Comment:
you will need to classify your test, either 'unit' or 'integration'. unit tests are standalone and don't needs any connection to the outside world and integration tests need a cluster connection or some other form of external communication, aka downloading things from the internet.

Without classifying the test it won't pass the linting check.

Reply:
from the build log:

```
Showing logs for build jenkins-x-jx-pr-6705-lint-hd552-94 stage ci and container step-ensure-test-classification
The following 1 test files are not classified with a valid Go build tag [unit|integration]
./hack/../pkg/cmd/opts/gocloud_integration_test.go
Pipeline failed on stage 'ci' : container 'step-ensure-test-classification'. The execution of the pipeline has stopped.
```